phpstan.neon.dist

```diff
@@ -1,8 +1,9 @@
 includes:
 	- phar://phpstan.phar/conf/bleedingEdge.neon
 parameters:
-	level: 4
+	level: 5
 	treatPhpDocTypesAsCertain: false
+	rememberPossiblyImpureFunctionValues: false
```

Comment:
I think it would be better if it had some doc reference on why this option was added?

Reply:
Oops. Merged before I saw this. Yes, good idea. Added in 0cd8565b7338cfa12753e70dc894902419db3805. I'll add a commit to https://github.com/WordPress/performance/pull/1203 to explain.

Reply:
Fixed in b86637e72644ff5e09d3932a87c3c2de8ae9baa5